ektelo/plans/hdmm_standalone.py

```diff
@@ -0,0 +1,614 @@
+from ektelo import support
```

Comment:
It looks like none of these plans are actually tested? Was the intention to replace calls to standalone.py in test plans?

Reply:
Yeah there is no need for the parallel implementation of hdmm plans, this file can be deleted